resources/assets/scss/app.scss

```diff
@@ -108,3 +108,7 @@ header.hero {
         }
     }
 }
+
+.w-max{
+    width: max-content;
+}
```

Comment:
This can be replaced by a configuration in the [tailwind.config.js](https://github.com/Astrotomic/astrotomic.info/blob/main/tailwind.config.js) and I would use `h-max-content` - just for clarity.
https://tailwindcss.com/docs/height#customizing

Reply:
It's not height problen, here I'm trying to fix that anchor width

![Screenshot from 2021-09-10 20-52-45](https://user-images.githubusercontent.com/60013703/132909935-c66f71bf-0625-4789-a116-174592249a62.png)

If we were working here with tailwind v2 I'll use just 'max-w-max' class and problem solved but here we just have tailwindcss v1.9.6 I don't think if there's a class for this!!

Reply:
Yeah, sorry - mixed height/width here. 🙈
We could possibly upgrade tailwind or add the same class via tailwind config. But I want to keep the custom CSS as small as possible.

Reply:
Fixed using tailwindcss v2.2.15